client/my-sites/upgrades/checkout-thank-you/jetpack-thank-you-card.jsx

```diff
@@ -0,0 +1,372 @@
+		} else if ( ! site.isMainNetworkSite() ) {
+			reason = translate( 'We can\'t install plugins on multisite sites.' );
+			this.trackConfigFinished( 'calypso_plans_autoconfig_error_multisite' );
+		} else if ( site.options.is_multi_network ) {
+			reason = translate( 'We can\'t install plugins on multi-network sites.' );
+			this.trackConfigFinished( 'calypso_plans_autoconfig_error_multinetwork' );
+		}
+
+		return (
+			<JetpackManageErrorPage
+				siteId={ this.props.siteId }
+				action={ translate( 'Contact Support' ) }
+				actionURL={ support.JETPACK_CONTACT_SUPPORT }
+				title={ translate( 'Oh no! We can\'t install plugins on this site.' ) }
+				line={ reason }
+				illustration={ '/calypso/images/jetpack/jetpack-manage.svg' } />
+		);
+	}
+
+	renderPlugin( key = 0, plugin ) {
+		const classes = classNames( 'checkout-thank-you__jetpack-plugin', {
+			'is-placeholder': ! plugin
+		} );
+		return (
+			<div key={ key } className={ classes } >
+				<div className="checkout-thank-you__jetpack-plugin-status-icon">
+					<span>
+						{ plugin
+							? this.getStatusIcon( plugin )
+							: this.getStatusIcon( { status: 'placeholder' } )
+						}
+					</span>
+				</div>
+				<div className="checkout-thank-you__jetpack-plugin-status-text">
+					<span>
+						{ plugin
+							? this.getStatusText( plugin )
+							: this.getStatusText( {
+								slug: 'placeholder',
+								status: 'placeholder'
+							} )
+						}
+					</span>
+				</div>
+			</div>
+		);
+	}
+
+	renderFeaturePlaceholders() {
+		const placeholderCount = !! this.props.whitelist ? 1 : 3;
+		return range( placeholderCount ).map( i => {
+			return this.renderPlugin( i );
+		} );
+	}
+
+	renderPlugins() {
+		const site = this.props.selectedSite;
+		let mappedPlugins;
+		if ( this.props.isRequesting || PluginsStore.isFetchingSite( site ) ) {
+			mappedPlugins = this.renderFeaturePlaceholders();
+		} else {
+			const plugins = this.addWporgDataToPlugins( this.props.plugins );
+			mappedPlugins = plugins.map( ( item ) => {
+				const plugin = Object.assign( {}, item, getPlugin( this.props.wporg, item.slug ) );
+				return this.renderPlugin( plugin.slug, plugin );
+			} );
+		}
+
+		return (
+			<div className="checkout-thank-you__jetpack-plugins">
+				{ mappedPlugins }
+			</div>
+		);
+	}
+
+	getStatusIcon( plugin ) {
+		switch ( plugin.status ) {
+			case 'done':
+				return <Gridicon icon="checkmark" size={ 18 } />;
+			case 'placeholder':
+				return 'x';
+			default:
+				return <Spinner size={ 18 } />;
+		}
+	}
+
+	getStatusText( plugin ) {
+		const { translate } = this.props;
+
+		if ( 'vaultpress' === plugin.slug ) {
+			switch ( plugin.status ) {
+				case 'done':
+					return translate( 'Backups and security active' );
+				default:
+					return translate( 'Activating backups and security' );
+			}
+		} else if ( 'akismet' === plugin.slug ) {
+			switch ( plugin.status ) {
+				case 'done':
+					return translate( 'Spam protection active' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **27** times:
`translate( 'Spam Protection' )` *ES Score: 9*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/first-pass-jetpack-plan-update)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).